bootc/ansible-facts/bootc.fact

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+BOOTC_SYSTEM="false"
+
+is_bootc() {
+  BOOTC_STATUS=$(sudo bootc status --json | jq .status.type)
```

Comment:
Could we not have written an ansible module to do this rather than writing it to the image itself?

Reply:
yes, that would also work